models/zeroex/optimism/zeroex_optimism_api_fills.sql

```diff
@@ -18,37 +20,35 @@ WITH zeroex_tx AS (
            max(affiliate_address) as affiliate_address
     FROM (
         SELECT tr.tx_hash,
-                    '0x' || CASE
-                                WHEN POSITION('869584cd' IN INPUT) <> 0
-                                THEN SUBSTRING(INPUT
-                                        FROM (position('869584cd' IN INPUT) + 32)
-                                        FOR 40)
-                                WHEN POSITION('fbc019a7' IN INPUT) <> 0
-                                THEN SUBSTRING(INPUT
-                                        FROM (position('fbc019a7' IN INPUT) + 32)
-                                        FOR 40)
-                            END AS affiliate_address
+                       CASE
+                            WHEN bytearray_position(INPUT, 0x869584cd ) <> 0 THEN SUBSTRING(INPUT
+                                                                                   FROM (bytearray_position(INPUT, 0x869584cd) + 16)
+                                                                                   FOR 20)
+                            WHEN bytearray_position(INPUT, 0xfbc019a7) <> 0 THEN SUBSTRING(INPUT
+                                                                                   FROM (bytearray_position(INPUT, 0xfbc019a7 ) + 16)
+                                                                                   FOR 20)
+                        END AS affiliate_address
         FROM {{ source('optimism', 'traces') }} tr
         WHERE tr.to IN (
-                -- exchange contract
-                '0x61935cbdd02287b511119ddb11aeb42f1593b7ef', 
+                 -- exchange contract
+                0x61935cbdd02287b511119ddb11aeb42f1593b7ef,
                 -- forwarder addresses
-                '0x6958f5e95332d93d21af0d7b9ca85b8212fee0a5',
-                '0x4aa817c6f383c8e8ae77301d18ce48efb16fd2be',
-                '0x4ef40d1bf0983899892946830abf99eca2dbc5ce', 
+                0x6958f5e95332d93d21af0d7b9ca85b8212fee0a5,
+                0x4aa817c6f383c8e8ae77301d18ce48efb16fd2be,
+                0x4ef40d1bf0983899892946830abf99eca2dbc5ce,
                 -- exchange proxy
-                '0xdef1abe32c034e558cdd535791643c58a13acc10'
+                0xdef1c0ded9bec7f1a1670819833240f027b25eff
```

Comment:
If we change this line back to `0xdef1abe32c034e558cdd535791643c58a13acc10`,
the `zeroex_tx` cte will have the tx_hash in the seed file:
https://dune.com/queries/3042243
I am not sure if this line is a typo, but I will change back for now.
```suggestion
                0xdef1abe32c034e558cdd535791643c58a13acc10
```